include/mtest.h

```diff
@@ -63,7 +63,7 @@
 #define REQUIRE_NE_STRING(expected, actual) REQUIRE_(strcmp((expected), (actual)),  "REQUIRE_NE_STRING(%s,%s) failed (\"%s\" == \"%s\")", #expected, #actual, expected, actual)
 
 // A test case is implemented as a function that returns 0 if the test passes and 1 if it fails.
-#define TEST_CASE(test_name, command...)                                                                                \
+#define TEST_CASE(test_name, command)                                                                                   \
```

Comment:
The variadic macro is used when the command contains a ','. See e.g. [test/test_mtest.c] (https://github.com/MortenSchou/mtest/blob/4fc43bfefd316439d6c914214f8c442c33f2149c/test/test_mtest.c#L10), which will fail to compile without the variadic macro. (This is why we write tests!! 😃 )

Reply:
Yeah, i dont know why it worked for me the first time around. Running the tests directly in the project made this also show to me. See the now changed code as i have added back in the variadic macros. (just not the named kind, which is GNU C only following the error in clang).